CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## [0.11]
+
+- **Refactored** Renderers - the `makeRenderer` functions in `server-render`
```

Comment:
re: serve all routes with `/admin` prefix: yes

re: changelog wording: now that I look back at the changes, none of the functions whose signatures changed (e.g. `getHtml`) are actually `export`ed, so no consumers could be using them anyway... I'll roll back the `breaking change` label + code - one of the things I was protecting against was a third party `import`ing something like `getHtml` and then needing to add `baseUrl` to the function call. But that's not possible.